lua/apisix/plugins/prometheus/exporter.lua

```diff
@@ -44,26 +47,35 @@ function _M.init()
     -- per service
     metrics.status = prometheus:counter("http_status",
             "HTTP status codes per service in Apisix",
-            {"code", "service"})
+            {"code", "service", "node"})
+
+    metrics.latency = prometheus:histogram("http_latency",
+        "HTTP request latency per service in Apisix",
```

Comment:
`APISIX` is better.

Reply:
You mean HTTP request latency should be changed to APISIX request latency?

Reply:
"HTTP request latency per service in Apisix" -> "HTTP request latency per service in APISIX"

Reply:
Okay. etcd_reachable/status/bandwidth also need to change Apisix to APISIX, as follows:
"Config server etcd reachable from Apisix, 0 is unreachable" -> "Config server etcd reachable from APISIX, 0 is unreachable"
"HTTP status codes per service in Apisix" -> "HTTP status codes per service in APISIX"
"Total bandwidth in bytes consumed per service in Apisix" -> "Total bandwidth in bytes consumed per service in APISIX"